src/data/valid/Database-extraction_set-minimal.yaml

```diff
@@ -3,7 +3,7 @@ extraction_set:
   - id: nmdc:extrp-99-abcdef
     name: DNA extraction of NEON sample WREF_072-O-20190618-COMP
     has_input:
-      - generic:xxx
+      - nmdc:procsm-99-123456
     #cardinality of outputs should be 1-many, may want to fractionate samples or keep supernatant
     has_output:
       - generic:yyy
```

Comment:
Same comments as on src/data/valid/Database-extraction_set-exhaustive.yaml, this example data will need to be updated once the range of has_output is fixed.